package.json

```diff
@@ -7,7 +7,9 @@
     "preansible": "./conf/preinstall.sh",
     "serverinstallbase": "./conf/serverinstallbase.sh",
     "serverinstallpm2": "ansible-playbook conf/pm2install.yml -i conf/inventory.ini --limit ${ENV_TO_DEPLOY:-dev}",
+    "copyexamples": "cp -nv conf/example-html/* frontend/ && cp -nv conf/example-assets/* conf/assets/ && cp -nv conf/example-stylesheets/*.scss frontend/stylesheets/",
```

Comment:
Fails if target file exits (exit code 1) which prevents the continuation of compilewithexamples script. We could do:

'cp -nv conf/example-html/* frontend/ || : && cp -nv ...'

That always returns 0. (And also masks all real errors.)